src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -21,18 +21,22 @@
 
 package ch.epfl.biop.imaris;
 
+
 import Ice.ObjectPrx;
 import Imaris.Error;
 import Imaris.*;
 import ImarisServer.IServerPrx;
 import com.bitplane.xt.IceClient;
+
 import ij.CompositeImage;
+import ij.IJ;
 import ij.ImagePlus;
 import ij.ImageStack;
 import ij.measure.Calibration;
 import ij.measure.ResultsTable;
 import ij.plugin.Duplicator;
 import ij.plugin.HyperStackConverter;
+import ij.plugin.Concatenator;
```

Comment:
Open question: Could we "just" use `import ij.plugin.*` seeing as we're using so many? Or do we want to explicitely import each one we want? Might make the code look a bit neater?

Reply:
no opinion. here is 3 a lot?

Reply:
Nah it's fine